src/stepper/Stepper.tsx

```diff
@@ -189,10 +220,16 @@ export class Stepper extends PureComponent<StepperProps, StepperState> {
     const isMin = this.state.value <= min
     const buttonSize = StepperButtonSize[display]
     const hasLeftAddon = display === StepperDisplay.SMALL && !isEmpty(leftAddon)
+    const isLeftAddonItinerary = hasLeftAddon && leftAddon.type === ItineraryType
 
     return (
-      <StyledStepper className={cc(['kirk-stepper', `kirk-stepper-${display}`, className])}>
-        {hasLeftAddon && <div className="kirk-stepper-left-addon">{leftAddon}</div>}
+      <StyledStepper
+        valueColor={this.getValueColor()}
```

Comment:
Correct if I'm wrong, but you could use this getValueColor() function on styled component declaration avoiding pass the value as props here

Reply:
Does that mean i'll have to pass the whole `threshold` and `max` props to the styled component and move the logic here?